lib/db/mem.js

```diff
@@ -817,6 +819,46 @@ module.exports = function (log, error) {
     return P.resolve({})
   }
 
+  Memory.prototype.createSecurityEvent = function (data) {
+    var addr = data.ipAddr
+    if (ip.isV4Format(addr)) {
+      addr = '::' + addr
+    }
+
+    var event = {
+      createdAt: Date.now(),
+      ipAddr: addr,
+      name: data.name,
+      uid: data.uid,
+      tokenId: data.tokenId
+    }
+    var key = event.uid.toString('hex')
+
+    var events = securityEvents[key] || (securityEvents[key] = [])
+    events.push(event)
+
+    return P.resolve({})
+  }
+
+  Memory.prototype.securityEvents = function (where) {
+    var key = where.uid.toString('hex')
+    var events = securityEvents[key] || []
+    var addr = where.ipAddr
+    if (ip.isV4Format(addr)) {
+      addr = '::' + addr
+    }
+
+    return P.resolve(events.filter(function (ev) {
+      return ev.uid.toString('hex') === key && ip.isEqual(ev.ipAddr, addr)
+    }).map(function (ev) {
+      return {
+        name: ev.name,
+        createdAt: ev.createdAt,
+        verified: (ev.tokenId && !!unverifiedTokens[ev.tokenId.toString('hex')])
```

Comment:
It did have the same problem, so I've updated the mem backend to act like the mysql.